src/components/DonorOverview.js

```diff
@@ -34,7 +36,7 @@ const DonorRecords = () => {
   )
 
   const CancelButton = () => (
-    <button onClick={()=>setEditing(false)} className="btn btn-outline-primary">Cancel</button>
+    <button onClick={()=>setEditing(false)} className="btn btn-link">Cancel</button>
```

Comment:
@Weiyuan-Lane Changed the Cancel button to btn-link to make the difference between 2 buttons clearer